clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -278,4 +282,4 @@ Average time from REGISTER to 200 OK: {} ms""".format(
     row['Retransmissions(C)'],
     rtt)
 
-print "\nLog files in /var/log/clearwater-sip-stress"
+print "\nLog files in /var/log/clearwater-sip-stress, with prefix {}".format(os.getpid())
```

Comment:
This relate to the prefix at line 60 (where we prefix our error logs with our PID) - I'll use LOG_PREFIX directly to make the link clearer.